api/v1alpha1/capacity_types.go

```diff
@@ -132,8 +132,21 @@ type CapacitySpec struct {
 	// The name style is similar to label keys. These are just names to link QCs
 	// together, and they are meaningless otherwise.
 	Cohort string `json:"cohort,omitempty"`
+
+	// QueueingStrategy indicates the queueing strategy of the workloads
+	// across the queues in this Capacity. This field is immutable.
```

Comment:
Document the meaning of the values here too. The constants don't form part of the API documentation.

Reply:
updated.